internal/db/p2p/p2p.go

```diff
@@ -40,6 +40,13 @@ var (
 	tracer = telemetry.NewTracer()
 )
 
+type (
```

Comment:
thought: I had no idea we could do this, although it makes sense from a syntax perspective :) Thanks :)